test/smallmatrix.jl

```diff
@@ -121,7 +121,54 @@ rmat = CrossRecurrenceMatrix(sparse(i,j,trues(length(i))))
     end
 end
 
-### Recurrence network
+### Skeletonization of RPs
+@testset "Skeletonized recurrence structures" begin
+    data = sin.(2*π .* (0:1000)./ 60)
```

Comment:
Here it is useful to start a 2nd test set named "sinusoidal"

Reply:
done